### DISCUSSION TITLE
Do the submission slideouts serve any real purpose?

### DISCUSSION THREAD

**Discussion by 1stevengrant** [solspace/craft-freeform#591] [Questions & Answers] [answered]
Shouldn't they contain the submission data

![](https://d.pr/i/Tw0ZAr+)

**Comment by kjmartens** [comment]
This might be a silly question, but how are you getting the slideout to happen? It's not expected and not intended to exist for Freeform, and this behavior doesn't happen for me...

**Reply by 1stevengrant** [reply]
when I visit the submissions page for a form and I double-click on the row (not the name). This is the quick editor that's available to Craft entries without redirecting to a full editor experience.

**Reply by kjmartens** [reply, marked as answer]
Oh, I see it now... I tried double-clicking before, but it seems to be a small sweet spot where it works. Depending on how that slideout data is handled, we might not be able to get submission data to display there since Freeform uses its own database tables to store the data. However, I've noted this for us to check into or see if we can disable this if not possible. 🙂

**Reply by 1stevengrant** [reply]
I've blown a few clients minds with that little feature
